### ISSUE TITLE
Great work!

### ISSUE THREAD

**Issue by ninjasort** [hoppula/refire#1]
This is really great. Just a few comments on the structure.

Thoughts on removing the components folder altogether and using something like this: http://marmelab.com/blog/2015/12/17/react-directory-structure.html? We know they are components at this point and for scaling up it makes sense to keep everything together by domain.

You could also split out the data bindings and group them into each of the domain-specific folders. For example you could put profile specific bindings in `/src/Profile/ProfileBindings.js`. This way you have a nice group of the following:

```
/src
  /Profile
    /Profile.js
    /Profile-spec.js
    /ProfileContainer.js
    /ProfileActions.js
    /ProfileReducer.js
    /ProfileReducer-spec.js
    /ProfileBindings.js
```

**Comment by hoppula** [comment]
Thanks!

I assume you mean the directory structure in [refire-forum repository](https://github.com/hoppula/refire-forum/tree/master/src/)? 

I agree that it would be best to move everything under `/src/components` to `/src` level. I've already placed most of related components under `/src/components/*`, e.g. all Thread related components at `/src/components/Thread`, but I agree that it'd make things simpler if all components and their related actions and reducers resided in their domain specific folders.

I've kept bindings in one file so far because it's easy to go through them quickly (they're like react-router routes) and avoid duplicating bindings. Your idea is good though, at some point they must be splitted to separate files.

I've also been thinking about the possibility to move bindings and routes to the folder with their related components, it would allow creating reusable mini-apps that could be utilized in any app. It would require some changes in refire, but it should be fairly quick to implement.
